osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs

```diff
@@ -56,6 +62,19 @@ public OsuPlayfield()
             hitPolicy = new OrderedHitPolicy(HitObjectContainer);
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            showPlayfieldBorder = config.GetBindable<bool>(OsuSetting.ShowPlayfieldBorder);
+            if (showPlayfieldBorder.Value)
+            {
+                AddInternal(new PlayfieldBorder
+                {
+                    RelativeSizeAxes = Axes.Both
+                });
+            }
```

Comment:
We aim to have all settings take effect immediately, even during gameplay. The way this is done makes it so that if the setting is changed after gameplay starts, it won't take effect until a restart.

As I said before, please add the border unconditionally (with zero alpha), and then register a `BindValueChanged(..., true);` callback in `loadComplete()` to toggle visibility of the playfield border upon any setting change.